src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs

```diff
@@ -885,26 +889,28 @@ private static long GetContentLengthFrom213Response(string responseString)
         private DateTime GetLastModifiedFrom213Response(string str)
         {
             DateTime dateTime = _lastModified;
-            string[] parsedList = str.Split(new char[] { ' ', '.' });
-            if (parsedList.Length < 2)
+            Span<Range> parts = stackalloc Range[3];
```

Comment:
Existing code only uses 3 parts, it looks like there will be no issue in case where were more parts in the string